lang/wasm_alien_thread_runner.cc

```diff
@@ -11,12 +11,15 @@
 #include <seastar/core/posix.hh>
 #include <seastar/core/reactor.hh>
 
+#include "log.hh"
 #include "lang/wasm.hh"
 #include "lang/wasm_alien_thread_runner.hh"
 #include "seastar/core/posix.hh"
 
 namespace wasm {
 
+static logging::logger alien_logger("alien");
```

Comment:
can we reuse the logger of `wasm`? IMHO, no need to have yet another logger just for this logging message, and `wasm` logger should serve this purpose well.